src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -10,11 +10,7 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 import seedu.address.commons.exceptions.IllegalValueException;
-import seedu.address.model.person.Address;
-import seedu.address.model.person.Email;
-import seedu.address.model.person.Name;
-import seedu.address.model.person.Person;
-import seedu.address.model.person.Phone;
+import seedu.address.model.person.*;
```

Comment:
I believe Intellij auto merge all the imports into the 1 wild card import statement. Probably need to ask Prof Damith about it.